### ISSUE TITLE
Validate and test certificate and key files before starting Tor2web software

### ISSUE THREAD

**Issue by fpietrosanti** [globaleaks/Tor2web#259]
This ticket is to Validate and test certificate and key files before starting Tor2web software, checking that keys and certificates are valid and can works.

In case one of the files is not valid an appropriate error shall be provided in the standard output and in the logs.

This should help troubleshooting errors such as:
Unhandled error in Deferred:
Unhandled Error
Traceback (most recent call last):
Failure: twisted.spread.pb.PBConnectionLost: [Failure instance: Traceback (failure with no frames): <class 'twisted.internet.error.ConnectionDone'>: Connection was closed cleanly.
]
Unhandled error in Deferred:
Unhandled Error
Traceback (most recent call last):
Failure: twisted.spread.pb.PBConnectionLost: [Failure instance: Traceback (failure with no frames): <class 'twisted.internet.error.ConnectionDone'>: Connection was closed cleanly.
]
Unhandled error in Deferred:
Unhandled Error
Traceback (most recent call last):
Failure: twisted.spread.pb.PBConnectionLost: [Failure instance: Traceback (failure with no frames): <class 'twisted.internet.error.ConnectionDone'>: Connection was closed cleanly.
]
